.storybook/routes.js

```diff
@@ -33,7 +33,7 @@ const routes = {
   '/components/popups/menu/': 'components-popups-menu--basic',
   '/components/popups/modal/': 'components-popups-modal--basic',
   '/components/popups/popup/': 'components-popups-popup--basic',
-  '/components/popups/toast/': 'components-popups-toast--with-popper',
+  '/components/popups/toast/': 'components-popups-toast--toast-dialog',
```

Comment:
Let's set the Toast route to point to the Basic example as is convention.

```suggestion
  '/components/popups/toast/': 'components-popups-toast--basic',
```